### PR TITLE
Update current challenge to Summer of Sexps

### PR DESCRIPTION
**Pull request by ErikSchierboom**
I'll leave the rest of the copy updating to you @ihid.

The track title doesn't overflow nicely for ClojureScript:

Screenshot 2023-05-28 at 09 06 38

### CODE REVIEW

File: app/commands/user/challenges/featured_exercises_progress_12_in_23.rb [hunk 1]
```diff
@@ -48,4 +49,7 @@ def featured_exercises
 
   MAY_TRACKS = %w[ballerina pharo-smalltalk prolog red rust tcl unison].freeze
   MAY_EXERCISES = %w[raindrops isogram roman-numerals space-age acronym].freeze
+
+  JUNE_TRACKS = %w[clojure clojurescript common-lisp emacs-lisp racket scheme].freeze
```
Comment (vaeng) [inline app/commands/user/challenges/featured_exercises_progress_12_in_23.rb]:
```suggestion
  JUNE_TRACKS = %w[clojure common-lisp emacs-lisp racket scheme].freeze
```

File: app/views/challenges/12in23.html.haml [hunk 2]
```diff
@@ -5,14 +5,14 @@
         .mr-auto{ class: 'max-w-[860px]' }
           .font-semibold.leading-150.flex.items-center.mb-4{ style: 'color: var(--colorPurpleToBrightPurple)' }
             %span.emoji.mr-6 📆
-            Month 5 of #12in23
-          %h1.text-h1.mb-8 It's Mindshifting May! ✨
+            Month 6 of #12in23
+          %h1.text-h1.mb-8 It's the Summer of Sexps! ✨
           %p.text-p-xlarge.mb-20
-            This month for #12in23 we're exploring languages that do things differently. We're digging into Ballerina, Pharo, Prolog, Red, Tcl and Unison, with interviews and live streaming, exclusive swag and lots of fun exercises for you to complete!
+            This month for #12in23 we're exploring languages that are Lisp dialects. We're digging into Clojure, ClojureScript, Common Lisp, Emacs Lisp, Racket and Scheme, with interviews and live streaming, exclusive swag and lots of fun exercises for you to complete!
```
Comment (vaeng) [inline app/views/challenges/12in23.html.haml]:
```suggestion
            This month for #12in23 we're exploring languages that are Lisp dialects. We're digging into Clojure, Common Lisp, Emacs Lisp, Racket and Scheme, with interviews and live streaming, exclusive swag and lots of fun exercises for you to complete!
```

File: app/views/challenges/12in23.html.haml [hunk 3]
```diff
@@ -25,13 +25,13 @@
...
           %p.text-p-large.mb-20
-            Earn the Mindshifting May badge for completing and publishing 5 exercises in one of the following languages.
+            Earn the Summer of Sexps badge for completing and publishing 5 exercises in one of the following languages.
             -#
               Consider starting with a language with a syllabus (
               = graphical_icon "concepts", css_class: 'w-[14px] h-[14px] filter-purple inline mx-[-4px]'
               ).
 
-          - tracks = Track.where(slug: %w[ballerina pharo-smalltalk prolog red tcl unison])
+          - tracks = Track.where(slug: %w[clojure clojurescript common-lisp emacs-lisp racket scheme])
```
Comment (vaeng) [inline app/views/challenges/12in23.html.haml]:
```suggestion
          - tracks = Track.where(slug: %w[clojure common-lisp emacs-lisp racket scheme])
```

**Review by vaeng — COMMENTED** [review]
With the clojurescript track gone, the overflow should not be a problem any longer, right?

**Comment by iHiD** [comment]
Thanks @vaeng :)

**Pull request merged**